docs/development/index.rst

```diff
@@ -0,0 +1,42 @@
+:orphan:
+
+***********************
+Contributing to Astropy
+***********************
+
+So you are interested in contributing to the Astropy Project?  Excellent! We
+love contributions! Astropy is open source, built on open source, and we'd
+love to have in our community.
+
+For the time being, please see our `CONTRIBUTING.md
+<https://github.com/astropy/astropy/blob/main/CONTRIBUTING.md>`_ document
+for an overview of ways you can contribute to Astropy.  The rest of this
+document is focused specifically on documentation for contributors and
+maintainers on how to work the Astropy repository for submitting
+documentation and code changes.
+
+For the guiding vision of this process and the project
+as a whole, see :doc:`development/vision`.
+
+.. toctree::
+   :maxdepth: 1
+
+   workflow/development_workflow
+   workflow/get_devel_version
+   when_to_rebase
+   codeguide
+   docguide
+   style-guide
+   testguide
+   ../testhelpers
+   scripts
+   building
+   ccython
+   releasing
+   workflow/maintainer_workflow
+   astropy-package-template
+   ../changelog
```

Comment:
Could we move the changelog link to the root page ? It was already hard to find in this section, if it's on another page it will be really hard. I like projects where I can find easily the changelog ;).

Reply:
I have also argued in the past for making the changelog easier to find.  I didn't like that it was buried in the "developer docs" since I think the full changelog can also be useful to users.  I'll see if I can find a new home for it as part of this reorg.

Reply:
New location seems perfect :+1: